src/main/java/uk/gov/companieshouse/filinghistory/api/service/Service.java

```diff
@@ -1,13 +1,22 @@
 package uk.gov.companieshouse.filinghistory.api.service;
 
+import java.util.List;
 import java.util.Optional;
-import uk.gov.companieshouse.filinghistory.api.model.FilingHistoryDocument;
+import uk.gov.companieshouse.filinghistory.api.model.mongo.FilingHistoryDocument;
+import uk.gov.companieshouse.filinghistory.api.model.mongo.FilingHistoryListAggregate;
 
 public interface Service {
 
+    Optional<FilingHistoryListAggregate> findCompanyFilingHistoryList(String companyNumber, int startIndex,
+            int itemsPerPage, List<String> categories);
+
```

Comment:
bunch of whitespace lines in here between the methods we could remove unless people think it impacts readability to do so.

Reply:
fair enough, I see you formatted the other interface methods to be consistent nice.